src/SequenceManager.cc

```diff
@@ -170,56 +169,14 @@ std::vector<SequenceManager::SequenceOnDisk> SequenceManager::list_sequences() c
 
         auto rel_path = std::filesystem::relative(entry.path(), path_);
```

Comment:
I would move this now into the following `if`'s scope, because it is only needed there.
The new removed `else` branch (i.e. suspicious folder found) that also needed `rel_path` is gone.